tests/e2e/steps_double_sign.go

```diff
@@ -128,3 +128,79 @@ func stepsDoubleSignOnProviderAndConsumer(consumerName string) []Step {
 		},
 	}
 }
+
+// Steps that make bob double sign on the consumer
+func stepsCauseDoubleSignOnConsumer(consumerName string) []Step {
```

Comment:
I think we should also take the provider chain as an argument here just to have it be more flexible

Reply:
Agree. Changed in [0b790649](https://github.com/cosmos/interchain-security/pull/1256/commits/0b790649a7ea28ea831e85bdb3cc57344517ea37).